packages/manager/apps/web/client/app/hosting/cdn/shared/leave-confirm-settings/confirm/hosting-shared-confirm-settings.html

```diff
@@ -1,91 +1,75 @@
 <oui-modal
-    data-heading="{{:: 'hosting_cdn_shared_modal_confirm_title' | translate}}"
-    data-primary-action="$ctrl.primaryAction()"
-    data-primary-label="{{'hosting_cdn_shared_modal_confirm_btn_validate' | translate}}"
-    data-secondary-action="$ctrl.goBack()"
-    data-secondary-label="{{:: 'hosting_cdn_shared_modal_confirm_btn_cancel' | translate}}"
+    heading="{{:: 'hosting_cdn_shared_modal_confirm_title' | translate }}"
+    primary-action="$ctrl.onConfirm()"
+    primary-label="{{:: 'hosting_cdn_shared_modal_confirm_btn_validate' | translate }}"
+    secondary-action="$ctrl.goBack()"
+    secondary-label="{{:: 'hosting_cdn_shared_modal_confirm_btn_cancel' | translate }}"
     type="info"
-    data-on-dismiss="$ctrl.goBack()"
+    on-dismiss="$ctrl.goBack()"
+    loading="$ctrl.loading"
 >
-    <!--Description-->
     <div
         class="oui-paragraph"
         data-ng-bind=":: 'hosting_cdn_shared_modal_confirm_info' | translate"
     ></div>
 
-    <!--Always online-->
-    <div class="mb-2">
-        <div class="row">
-            <div class="text-right col-md-6">
-                <p class="oui-heading_5 mb-1"><strong>Always online</strong></p>
-            </div>
-            <div class="text-left col-md-6">
-                <p
-                    class="mb-1"
-                    data-ng-bind="::'hosting_cdn_shared_state_' + ($ctrl.model.alwaysOnline.enabled ? 'enable' : 'disabled') | translate"
-                ></p>
-            </div>
+    <dl class="oui-description oui-description_horizontal">
+        <dt><strong>Always online</strong></dt>
```

Comment:
I see a lot of text hardcoded in the HTML file directly. The translation is not required?

Reply:
Same than for other pages, all direct texts in HTML templates won't be translated.